pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/ProvisioningTimeoutModelDefTest.java

```diff
@@ -1,69 +0,0 @@
-/*
- * The MIT License
- *
- * Copyright (c) 2016, CloudBees, Inc.
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- */
-package org.jenkinsci.plugins.pipeline.modeldefinition;
-
-import hudson.model.Result;
-import hudson.model.Slave;
-import org.junit.BeforeClass;
-import org.junit.ClassRule;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-import org.jvnet.hudson.test.BuildWatcher;
-import org.jvnet.hudson.test.Issue;
-
-/**
- * @author Jose Taboada
- *
- * The goal of this test is check the nuances between top level and stage level agents regarding timeouts (options)
- *  Top Level Agents: timeout is only for the build itself, therefore won't fail.
- *  Stage Level Agents: timeout includes provisioning, in this example is +600ms
- *
- * Jira Ticket JENKINS-60994 to describe this problem.
- */
-@RunWith(Parameterized.class)
-public class ProvisioningTimeoutModelDefTest extends AbstractModelDefTest {
-
-    @ClassRule
-    public static BuildWatcher buildWatcher = new BuildWatcher();
-
-    @Parameterized.Parameters
-    public static Object[][] data() {
-        return new Object[10][0];
-    }
-
-    @Test
-    @Issue("JENKINS-60994")
-    public void topLevelAgent() throws Exception {
-        expect("options/topLevelAgentSuccessWithTimeout")
```

Comment:
Should the script files be deleted as well, or are they still used by other tests?

Reply:
This is what irritates me about the idiom of keeping test pipelines as separate files: syntax highlighting etc. are certainly nicer than when inline as Java strings, but it is harder to review diffs in some cases and you can wind up with stale files.